packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -82,62 +91,62 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
             "& $carousel_flipperPrevious, & $carousel_flipperNext": {
                 color: "black",
                 fill: "black",
```

Comment:
Should this be pure black? Most cases we don't use black we use `#101010`

Reply:
@bheston for guidance

Reply:
Is that the foreground color for the chevron icons? If so, it should be #101010. I know flippers might be a little different because of opacity, but I think most of the fills and outlines should be some other non-black color on the neutral color ramp. The only thing that is currently full black is text over light-colored accent buttons.

Reply:
Ok, changed all black to `#101010`